modules/@apostrophecms/schema/ui/apos/logic/AposInputRelationship.js

```diff
@@ -81,6 +81,13 @@ export default {
       };
     },
     chooserComponent () {
+      const isPage = apos.modules['@apostrophecms/page'].validPageTypes
+        .includes(this.field.withType);
+
+      if (isPage) {
+        return 'AposDocsManager';
+      }
+
```

Comment:
this block says "hey you are a page type, so I'm giving you the `AposDocsManager` instead of the regular `AposPagesManager`

Reply:
This is good, but take a look at my comment earlier about arranging for `components` to get set up properly in `getBrowserData`, I think you would be able to remove this code in favor of letting it default nicely to `AposDocsManager` like it does for pieces & also be overrideable per-type like it is for pieces (certainly an unusual move for a page type, but since we can already do it)

Reply:
Much better!